### PR TITLE
feat: beacon permissions

### PR DESCRIPTION
**Pull request by harmbrugge**
What are the main changes you did:
Added three permission levels secure the beacon responses BOOLEAN, COUNT and RANGE :
Exists (boolean yes/no response to queries)
Range (inexact number of results found)
Count (exact number of counts)

Other permissions: if you have view permission you always get the exact count as an aggregator you only get an exact match if user only if counts >= 10

Step size for range queries are currently hardcoded at 10.

todo:
- [x] updated docs in case of new feature
- [x] added/updated tests
- [x] added/updated testplan to include a test for this fix, including ref to bug using # notation

### CODE REVIEW

File: backend/molgenis-emx2-beacon-v2/src/main/java/org/molgenis/emx2/beaconv2/endpoints/EntryTypes.java [hunk 1]
```diff
@@ -11,20 +11,20 @@
 import org.molgenis.emx2.beaconv2.BeaconSpec;
 import org.molgenis.emx2.beaconv2.EntryType;
 import spark.Request;
```
Comment (mswertz) [inline backend/molgenis-emx2-beacon-v2/src/main/java/org/molgenis/emx2/beaconv2/endpoints/EntryTypes.java:13]: principieel zou je liefst de Request/Response niet nodig hebben hier. Want nu heb je onnodige dependency. Liefst pass je wat je nodig hebt. Kan dat? Of is dat teveel nu (je zit met oude code en dus even zo houden dan maar?)

**Review by mswertz — APPROVED** [review]
if Joeri approves functionally then you are ready to go.

**Review by joerivandervelde — CHANGES_REQUESTED** [review]
Looks great, but I would like to request an additional 'count' permission to allow fine-grained discovery for rare disease use cases.

**Comment by harmbrugge** [comment]
> Looks great, but I would like to request an additional 'count' permission to allow fine-grained discovery for rare disease use cases.

Agreed: added a count user allowing to see count queries with results < 10

**Review by joerivandervelde — APPROVED** [review]
Great to also have COUNT :)

**Pull request merged**